src/linker/configuration.py

```diff
@@ -321,3 +326,18 @@ def _validate_environment(self) -> Dict[Any, Any]:
             )
 
         return errors
+
+    def _copy_configuration_files_to_results_directory(
+        self,
+        pipeline_specification_path,
+        input_data_specification_path,
+        computing_environment_specification_path,
+    ) -> None:
+        _ = os.umask(0o002)
```

Comment:
Whoa. Does this get set back?

Reply:
Hmm, good catch, it doesn't.

Reply:
Added a line to set it back at the end now